Lesson1/MyPlayground.playground/Contents.swift

```diff
@@ -1,2 +1,73 @@
-import UIKit
+var inputData: [Int] = [1, 2, 4, 1]
 
+/// The mean for an array of integers
+/// - Parameter inputData:The array of integers with an amount greater than 2
+/// - Returns: The mean; otherwise, nil.
+func mean(for inputData: [Int]) -> Double? {
+    if inputData.count < 2 { return nil }
+    var sum = 0
+    let count = inputData.count
```

Comment:
nit: do you need a separate constant `count`? 
We use `count` only once in our code, additionally it does not add any additional information in comparison with `inputData.count`. I would use `inputData.count` below:

```swift
let mean = Double(sum) / Double(inputData.count)
```

Reply:
fixed